### DISCUSSION TITLE
updating my skills on linkedin

### DISCUSSION THREAD

**Discussion by sadique009** [github/india#643] [General]
I wanted to know how to update my Microsoft azure skills on linkedin as I am unable to do so.

**Comment by kitarp29** [comment]
There are multiple ways to do it.
1. Go to *Skills & endorsements* section, choose *Add a new skill* 
2. If you have some project or experience regarding it. Update in the respective sections.
3. If you have certifications, update at *Licenses & certifications*

I hope this helps.
